frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx

```diff
@@ -90,17 +107,17 @@ export const HTTPRequestTypeForm: React.FC<RequestTypeFormProps> = ({ ports, pro
         label="Path"
         placeholder="/"
       />
-      {renderPortField(ports, `healthChecks.${probeType}.data.httpGet.port`, port)}
+      {renderPortField(ports, `healthChecks.${probeType}.data.httpGet.port`, port, resources)}
```

Comment:
You should have param names as similar as you have in the function declaration. Otherwise its difficult to understand what values you are sending to a certain function.
```suggestion
      {renderPortField(ports, `healthChecks.${probeType}.data.httpGet.portName`, defaultPort, resourceType)}
```